src/components/AboutUsIntroduction/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+import Trans from 'next-translate/Trans';
+
+const AboutUsIntroduction = () => {
+  const { t } = useTranslation('about-us')
+  const Component = (props) => <h1 className="about-us-introduction__title" {...props} />
```

Comment:
How about renaming `Component` to `IntroductionTitle`?

Reply:
Since this is the function returning an element, should we go with `renderIntroductionTitle`

Reply:
@LilBank I agree, `Component` is too general. Anyway, Can I use just only `Title` since this component only focuses on `AboutUsIntroduction`, what do you think?

Reply:
I think @kimvcp 's way is interesting to approach. I'll try as Kim said in the first place because I've never done it before.